resources/scripts/components/elements/DropdownMenu.tsx

```diff
@@ -119,14 +119,14 @@ const DropdownMenuRadioItem = React.forwardRef<
     <DropdownMenuPrimitive.RadioItem
         ref={ref}
         className={cn(
-            'relative flex cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none transition focus:duration-0 focus:bg-[#ffffff33] focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50',
+            'relative flex cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none transition focus:duration-0 focus:bg-[#ffffff10] focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50',
             className,
         )}
         {...props}
     >
         <span className='absolute left-2 flex h-3.5 w-3.5 items-center justify-center'>
             <DropdownMenuPrimitive.ItemIndicator>
-                <DotFilledIcon className='h-4 w-4 fill-current' />
+                <a className='text-neutral-500'>{`>`}</a>
```

Comment:
Not sure about this being an anchor tag. Make it a div or something

Reply:
yea i missed this should prob be div